bsp/esp32_p4_function_ev_board/README.md

```diff
@@ -15,13 +15,24 @@ ESP32-P4 Function EV Board is internal Espressif board for testing features on E
 Configuration in `menuconfig`.
 
 Selection LCD display `Board Support Package(ESP32-P4) --> Display --> Select LCD type`
-- LCD 7-inch 1280x800 - ili9881c (default)
+- LCD 1280x800 - ili9881c (default)
 - LCD 1024x600 - ek79007
+- HDMI - lt8912b
+    - 800x600@60HZ
+    - 1280x720@60HZ
+    - 1280x800@60HZ
+    - 1920x1080@30HZ
 
 Selection color format `Board Support Package(ESP32-P4) --> Display --> Select LCD color format`
 - RGB565 (default)
 - RGB888
 
+## HDMI Support
+
+This BSP supports HDMI converter Lontium LT8912B. Follow these rules for using it with HDMI:
+- Use ESP-IDF 5.4 or older (from commit [93fdbf2](https://github.com/espressif/esp-idf/commit/93fdbf25b3ea7e44d1f519ed61050847dcc8a076))
+- Only RGB888 is supported with HDMI
+- Use MIPI-DSI to HDMI converter from this link ([Schematic](), [PCB]())
```

Comment:
Please add the links :)

Reply:
Sorry, I forgot remove it. I still haven't got permission for sharing it.